docs/how-to/understanding-break-the-monolith.md

```diff
@@ -2,45 +2,59 @@
+#### Example
 
-### 2. Redesign and Breaking Services
+If an entity in your original monolith service has a non-relation field like `createdAt` (a date field), this field will appear in the new service with the same properties, ensuring data consistency.
 
-In the Redesign mode, you can manually or with the assistance of AI, restructure your monolith or individual services. This includes the ability to:
+### Handling One-to-One Relations
 
-- Move entities between existing or new services.
-- Utilize the Intelligent AI Helper for suggestions on optimal microservice configurations.
-- Create tailored micro-services based on the suggested architecture.
+In one-to-one relationships, both entities involved in the relationship retain a single field representing the remote entity's ID. This approach maintains a link between the entities even after they are distributed across different services.
 
-The AI Helper identifies logical groupings of entities based on their relationships and usage patterns. It also suggests ways to cut relations between entities beyond group boundaries.
+#### Example
 
-:::note
-**The AI Helper uses advanced LLMs for insightful microservice architecture suggestions**, analyzing your entity details with a commitment to high privacy standards.
-:::
+Each entity will have a field, like `partnerId` in a `Customer` entity, which points to the corresponding entity in the other service. This mutual referencing ensures that the entities can still reference each other post-transition.
 
-### 3. Applying the New Architecture
+### Handling One-to-Many Relations
 
-Once you are satisfied with the new design, "Break The Monolith" facilitates the creation of new microservices complete with their entities and APIs that bridge the entity relations separated across services, transforming foreign keys into remote IDs where necessary.
+In a one-to-many relationships, the handling of fields is nuanced:
 
-This process ensures that your code is production-ready and supports the new microservice architecture.
+#### The "Many" Side
+
+The entity on the "many" side of the relationship, upon being moved, retains a field representing the ID of the "one" side. This field, like `customerId` in an `Order` entity, is an actual field in the database. It allows the service to reference the related entity in another service, preserving the integrity of the relationship even when the entities are separated.
+
+#### The "One" Side
+
+On the "one" side of the relationship, no direct relational fields are created to represent the "many" side. This side, therefore, loses the direct link to the "many" entities, reflecting the separation of services.
+
+#### Example
+
+For example, if a `Customer` entity (one side) and `Order` entities (many side) are separated, the `Order` entities will retain a `customerId` field to refer to their corresponding `Customer`. However, the `Customer` entity won't have a direct reference to its orders in the new structure.
+
```

Comment:
Let's mention also the many-to-many option that is not handled as part of the Beta, and no related fields are created at both sides.